Runtime/JSObject.cs

```diff
@@ -0,0 +1,41 @@
+using System.Collections;
+using System.Collections.Generic;
+
+namespace NodeApi;
+
+public struct JSObject : IEnumerable<(JSValue name, JSValue value)>, IEnumerable
```

Comment:
Should it be `IDictionary<JSValue, JSValue>`?

Reply:
Good point! I can add it in a follow up PR. Here I was focused only on `IEnumerable`. It would be still required for `IDictionary`. I just need to change the tuple to `KeyValuePair`.

Reply:
I have added the `IDictionary<JSValue, JSValue>`. Some properties/methods are not implemented yet.